### PR TITLE
Add "minutes exempted" field to Attendance Day

### PR DESCRIPTION
**Pull request by aaronbrowder**
Adding "minutes exempted" to the Off Campus Time feature. This adds an extra column when editing/viewing off campus time. 

Normally off campus time is not counted towards the student's attendance for the day. When minutes exempted is set, that number of minutes of the off campus time WILL be counted towards the student's attendance for the day.

### CODE REVIEW

**Comment by schmave** [comment]
Is "minutes exempted" the best name for this? Seems like "attendance minutes" or "minutes included in the school day" or something positive like that might be clearer. But I'm not the one using it, so I will defer to your preference :)

**Comment by schmave** [comment]
Just let me know what you think.

**Comment by aaronbrowder** [comment]
Yeah, it's not the most clear name, but I don't think there's a clearer one
given that it needs to be short enough to fit in the table headers.

**Comment by aaronbrowder** [comment]
Any estimate of when this might be merged?

**Comment by schmave** [comment]
Sorry, some things have come up and I got distracted from this. I'll try to
merge today.

Evan

File: app/views/attendance_person.scala.html [hunk 1]
```diff
@@ -85,6 +85,7 @@ <h3>Details</h3>
 	@if( has_off_campus_time ) {
 		<th width=75>Off Campus Start Time</th>
 		<th width=75>Off Campus End Time</th>
+		<th width=75>Minutes Exempted</th>
```
Comment (schmave) [inline app/views/attendance_person.scala.html:88]: This, and the associated &lt;td> should only be shown if the organization has enabled logging of off-campus time.

Comment (schmave) [reply app/views/attendance_person.scala.html:88]: Sorry, I'm a dummy. I see that you already did this.

**Comment by schmave** [comment]
This is live on the site now. Thanks for your patience and your contribution!

**Comment by aaronbrowder** [comment]
Thank you!

**Pull request merged**